### ISSUE TITLE
ZIP video file

### ISSUE THREAD

**Issue by nttri** [marmelroy/Zip#182]
HI team, can I use this library to zip a video file with .mov/.mp4 format ?

**Comment by CraigSiemens** [comment]
Yes, but videos are already in a fairly compressed format so you won't get much of a size reduction.
